packages/sit-onyx/src/components/OnyxDataGrid/features/sorting/sorting.ct.tsx

```diff
@@ -0,0 +1,73 @@
+import type { Locator } from "@playwright/test";
+import { expect, test } from "../../../../playwright/a11y";
+import SortingDataGrid from "../../../examples/DataGrid/SortingDataGrid.vue";
+
+const getTestArray = () => [
```

Comment:
```suggestion
const getTestData = () => [
```

Reply:
Same for the other files :) reduced cognitive overhead since the return value is the `data` property of the data grid :)